### ISSUE TITLE
[Issue] PiFBA issue in PI Zero with latest pre build image 

### ISSUE THREAD

**Issue by makemao** [kiteretro/Super-AIO#6]
Hello there,

I built my GBZ using your latest SAIO v0.6c board.
I used pre build image retropie-4.3-rpi1_zero_SAIO_20171002-211534.zip

All is working as expected but the PiFBA emulator who is behaving too fast in fact, basically all ROMs from CPS1 to neogeo are running too fast & sound is also accelarated. For info I tried to use Binary and also use source, both 'methods' have same issue.

Any clue on this problem ?

Regards.

**Comment by geebles** [comment]
Hi, no idea off the top of my head but it's nothing to do with anything I've done, as this will affect any retropie/emulationstation/retroarch on a Pi :) I've literally changed no settings other than the battery overlay!

Take a look here and try what the comments say: https://www.reddit.com/r/RetroPie/comments/4fnk79/emulators_running_way_too_fast/

Probably a good idea to see if it does it on STOCK retropie.. and then ask specifically on a retropie forum!

Could you let me know specifically the name of the ROM and what BIOS you are using? Do not link anything, it will be deleted if so

**Comment by ltpitt** [comment]
I am not sure this is related but I have the same issue with sound speed on PICO-8.
I've also asked in their specific forum:
https://www.lexaloffle.com/bbs/?tid=30656

In case of news I will update

**Comment by geebles** [comment]
Thanks please keep us posted. I will attempt to reproduce issue my end and see if I can find a fix but it won't be for a little while

**Comment by ltpitt** [comment]
Hi all just a minor update, I do not know if it can be useful or not to narrow down the issue.

- Stock Retropie on Raspberry 3 has no audio issues using PICO-8
- I've installed PICO-8 also on a Raspiboy and even there I do not have the ultra speed audio issue

I am loving my SuperAIO we basically live together.

I hope to get a CS soon :)

**Comment by ltpitt** [comment]
In PiFBA Raspberry port [README.md](https://github.com/RetroPie/pifba/blob/master/readme.txt) I read only this information related to audio:

_If you're having problems with HDMI audio then it is likely PulseAudio is causing the issues as it has problems with the ALSA drivers. To fix this simply remove PulseAudio:

* sudo apt-get --purge remove pulseaudio
* sudo apt-get autoremove_

I do not see how this could be related but before testing it I'll have to backup my SD.

**Comment by ltpitt** [comment]
I've also find an interesting workaround.

The issue here is run Neogeo titles at full speed with good sound so I think that GnGeo-Pi could be an interesting solution too.

I need to investigate how to set GnGeo-Pi controls for SAIO.

Do you guys have any pointer?

**Comment by ltpitt** [comment]
Hi @kiteretro !

Would you be so kind - if and when you have time - to give me some suggestion to proceed in any direction to have NeoGeo on SuperAIO?

Both PiFba or Gneopi solutions would be fine for me but for pifba I cannot find out why it is so fast and for gneopi I cannot configure keys.

Your time and help would be greatly appreciated :)

**Comment by makemao** [comment]
Hello Guys,

After au pause I decided to spend some time this week end on the pifba running too fast issue.
I finally found something that fix the issue.
I had to edit the /boot/config.txt and lower down the pixel frequency to 64 instead of 96:

hdmi_timings=320 1 20 30 38 240 1 4 3 10 0 0 0 60 0 **6400000** 1

It works perfectly now and I finally have smooth neogeo and cpsx arcade game on pi zero :)

Time to build the gbz with sword circuit now.

Regards.
Yann
